packages/migration-data/bin/cli.ts

```diff
@@ -37,7 +37,10 @@ program
       }
     }
 
-    fs.writeFileSync('./data/evm-addresses.json', JSON.stringify(addrs, null, 2))
+    fs.writeFileSync(
+      './data/evm-addresses.json',
+      JSON.stringify(addrs, null, 2)
+    )
```

Comment:
Happened when I ran `yarn lint:fix`. I guess we're not linting this package in CI.